en/developer/10-scm-plugins.md

```diff
@@ -60,13 +60,12 @@ file using the following patterns:
 * `action.ID.property.NAME.description` Description for property named "NAME" for action view with ID "ID"
 
 
-[Plugin Development - Plugin Localization]: plugin-development.html#plugin-localization
-[codes]: plugin-development.html#defining-plugin-localization-messages
+[codes][page:developer/01-plugin-development.md#defining-plugin-localization-messages]
```

Comment:
Again, this link reference definition was converted to an actual link, when instead the referring link earlier in the document (line 49) should be updated instead, and this line (63) removed

Reply:
fixed...